themes/theme-b2b/src/index.ts

```diff
@@ -377,6 +384,13 @@ const theme: BaseTheme = {
       lineHeight: 'body',
       color: colors.gray30,
     },
+    section: {
+      fontFamily: 'body',
+      fontSize: 'small',
+      fontWeight: 'body',
+      lineHeight: '32px',
```

Comment:
why don't you use lineHeight from theme?

Reply:
line height from theme is just:
`lineHeights: {
    body: 1.5,
    heading: 1.125,
  },`

32px like in the figma design is 2rem

Reply:
We actually use 32px lineHeight a lot in the theme, maybe we should think about adding 1.5rem and 2rem to the theme, but not in this issue :)

Reply:
yes good idea!